eng/common/core-templates/steps/source-index-stage1-publish.yml

```diff
@@ -1,6 +1,6 @@
 parameters:
-  sourceIndexUploadPackageVersion: 2.0.0-20250818.1
-  sourceIndexProcessBinlogPackageVersion: 1.0.1-20250818.1
+  sourceIndexUploadPackageVersion: 2.0.0-20250906.1
+  sourceIndexProcessBinlogPackageVersion: 1.0.1-20250906.1
```

Comment:
I'd like to take this PR to see if [my source-indexer fix](https://github.com/dotnet/source-indexer/pull/193) works in Roslyn. So I will revert the SDK bump and merge it, hope that's alright...